app/models/health/patient_referral.rb

```diff
@@ -313,8 +307,26 @@ def record_status
       end
     end
 
-    def disenrolled?
```

Comment:
Should the remaining `disenrolled?` be updated to use `actual_or_pending_disenrollment_date.present?`

Reply:
Just pushed an update to use it.